en/developer_guide/logs.rst

```diff
@@ -0,0 +1,90 @@
+Logs
+====
+
+Our backoffice logging is based on Monolog.
+We provide an interface to display all the user action on the website
+It displays the date and time, user's IP, user's name, the current site's name and the log's message.
```

Comment:
I think this sentence is too much, the one before can only invite the user to take a look at it.

Maybe you can add a screen shot of some lines